index.html

```diff
@@ -82,7 +82,7 @@
             </svg>
 
             <h1 id="title-text">billboard.js</h1>
-            <h4>Re-usable easy interface JavaScript chart library, based on D3 v4+.</h4>
+            <h4>Re-usable, simple interface JavaScript chart library, based on D3 v4+.</h4>
```

Comment:
We used `easy` and this meaning is one of the characteristic functionality of the library.
I think other corrections are good, except this. If you can let as before, we can merge your changes.

and thanks for your contribution!